src/annotator/anchoring/text-range.ts

```diff
@@ -107,12 +109,11 @@ export class TextPosition {
 
   /**
    * Return a copy of this position with offset relative to a given ancestor
-   * element.
+   * element .
```

Comment:
Whoops.

Reply:
? - I assume you intended to remove this extra space.

Reply:
Yep. I left the comment to remind myself.